.github/linters/.arm-ttk.psd1

```diff
@@ -4,7 +4,7 @@
 @{
     # Test = @( )
     Skip = @(
-        'Template Should Not Contain Blanks',
+        'Template Should Not Contain Blanks'
```

Comment:
Change "Should" to "Must"

Reply:
This is pre defined. This cannot be changed.